Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ghcr.io/plc-lang/rust-llvm:latest
+
+# Give all users access to cargo and rust home
+RUN chmod -R a+rw $CARGO_HOME \
+    && chmod -R a+rw $RUSTUP_HOME
+
+# Required if we want to use `lld` as the default linker for RuSTy
+ENV LLVM_VER=14
+RUN ln -sf /usr/bin/ld.lld-$LLVM_VER /usr/bin/ld.lld
+
+# Install the local RuSTy version
+WORKDIR /rusty
+COPY . .
+RUN ./scripts/build.sh --build
```

Comment:
wouldn't it make more sense for this to be release? I also wonder if we should also include the standard functions here.

Reply:
Yes, I'll update it to release. 

Where were you thinking to place the standard functions?